packages/web-pixels-extension/src/types/index.ts

```diff
@@ -1,59 +1,8 @@
 export type {ExtensionApi} from './ExtensionApi';
 export type {RegisterInit, InitData} from './RegisterInit';
 export type {PublicApi} from './PublicApi';
-export type {
-  Attribute,
-  PixelEvents,
-  Browser,
-  Context,
-  Cart,
-  Shop,
-  CartLine,
-  Customer,
-  DiscountApplication,
-  GenericElement,
-  InputElement,
-  FormElement,
-  Product,
-  Collection,
-  Checkout,
-  CheckoutLineItem,
-  CartCost,
-  CartLineCost,
-  OrderCustomer,
-  Order,
-  MailingAddress,
-  ProductVariant,
-  DiscountAllocation,
-  PixelEventsProductAddedToCartData,
-  PixelEventsCheckoutStarted,
-  PixelEventsCheckoutCompleted,
-  PixelEventsCheckoutStartedData,
-  PixelEventsCheckoutCompletedData,
-  PricingPercentageValue,
-  Transaction,
-} from './PixelEvents';
-export type {
-  PublisherData,
-  PublisherCustomData,
-  PublisherOptions,
-  SubscriberCallback,
-  SubscriberOptions,
-} from './EventBus';
-export type {
-  StandardEvents,
-  CustomEvents,
-  DomEvents,
-  DomEventName,
-  PixelEventName,
-  StandardEventName,
-  Events,
-  SchemaVersion,
-  WebPixelsManager,
-} from './WebPixelsManager';
-export type {
-  CustomerPrivacyEventBus,
-  ConsentEvents,
-  ConsentTrackingEventName,
-  CustomerPrivacyPayload,
-} from './PrivacyApi';
+export type * from './PixelEvents';
+export type * from './EventBus';
+export type * from './WebPixelsManager';
+export type * from './PrivacyApi';
+export type * from './VisitorApi';
```

Comment:
I've always discarded all these changes. I'm not sure why it changes the exporting to this, or what the team's preference is.